### ISSUE TITLE
[Docker]: Look into the possibility of building the image in Docker container

### ISSUE THREAD

**Issue by samip5** [GenPi64/Build.Dist#111]
We should probably look into if we could actually leverage Docker with building the images.

**Comment by jonesmz** [comment]
I certainly agree that being able to provide a docker image that can build the genpi64 image would be a value add. But I don't think it would be good to make that the only build path, since I don't run docker on any of my systems.

**Comment by samip5** [comment]
I agree with that, and my main intention with this is that it would help if we could just leverage Docker for automated building of the image for example.

**Comment by jonesmz** [comment]
Sounds good!
